docs/source/technical_tutorials/dps_tutorial/DPS_runner_template.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>This probably belongs in a different place - refer to Registering an Algorithm section. This section should just say: Make sure you are using an already registered algorithm or register one (link).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>